java/client/src/main/java/glide/api/commands/StringCommands.java

```diff
@@ -0,0 +1,47 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.commands;
+
+import glide.api.models.Ok;
+import glide.api.models.commands.SetOptions;
+import glide.api.models.commands.SetOptions.ConditionalSet;
+import glide.api.models.commands.SetOptions.SetOptionsBuilder;
+import java.util.concurrent.CompletableFuture;
+
+/** String Commands interface to handle single commands that return Strings. */
```

Comment:
They do not have to return strings. "Commands that perform string manipulation." this could be better and we could add a link to it https://redis.io/commands/?group=string

Reply:
Good point.  Have you updated this in the string commands PR?  I can grab what you have there...

Reply:
This is what I had. (It is not fully up to date that PR)
`/**
 * Commands that perform string manipulation.
 *
 * @see: <a href="https://redis.io/commands/?group=string">String Commands</a>
 */`

Reply:
I removed the "that return Strings":  
```
/**
 * String Commands interface to handle single commands.
 * 
 * @see <a href="https://redis.io/commands/?group=string">String Commands</a>
 */
```